src/main/java/hw6_jdi/jdiPages/HeaderSection.java

```diff
@@ -0,0 +1,47 @@
+package hw6_jdi.jdiPages;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Icon;
+
+public class HeaderSection extends Section {
+
+    @FindBy(xpath = "//ul[@class='uui-navigation nav navbar-nav m-l8']/li[1]/a")
```

Comment:
I suppose that this can be simplified

Reply:
Menu element was added (instead of 4 elements & 4 methods)